recipes/wurst/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "wurst" %}
+{% set version = "0.3.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ddb56c990945a73e42af3773c712e6ca144244d4584b144d0777a309ffe4e40d
+
+build:
```

Comment:
You need to lis[t entry points ](https://github.com/polca/wurst/blob/a30a6ebefb5584edb06ce2eff742b2e566e49ae0/setup.py#L28)in the conda-recipe. Please check the conda-forge docs section on noarch: python.